opensrp-chw-core/src/main/res/layout-w900dp/navigation_view.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@LZRS Move this to CHW and make it Exclusive to LMH flavor

Reply:
https://github.com/opensrp/opensrp-client-chw/pull/1927